### ISSUE TITLE
Support postgres: insert row return error

### ISSUE THREAD

**Issue by visteras** [golobby/orm#13]
In readmy I see that postgres is supported, but just `insert row` is returned like this: `LastInsertId is not supported by this driver` (used by pq)

A simple google search for me: https://stackoverflow.com/questions/33382981/go-how-to-get-last-insert-id-on-postgresql-with-namedexec

I think for postgres the insert query needs to be changed for the used PK return... thoughts?

**Comment by visteras** [comment]
P.S. And if i use uuid as PK a cant used genericSet because i give panic... i think need used set value with other variant(but i dont known how...), because non any type we can set from reflect value set...

**Comment by amirrezaask** [comment]
yes I can add support for `RETURNING` when using postgres and thing about PK is true now we are just supporting I think int but it's just a line change to add other ones, maybe you can do both PRs yourself and become contributer, I would be glad, I can tell you where to look
